internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -79,15 +79,9 @@ func setCPUSLoadBalancing(c *oci.Container, enable bool, schedDomainDir string)
 			if err != nil {
 				return err
 			}
-
-			if path == cpuSchedDomainDir {
```

Comment:
how is this redundant?

Reply:
the next `if info.Name() != "flags" ` will catch it anyway